Bottom: 7418ac143864a4ab58c53180f75346365d02c002
Top:    0c12f7a76216b187f4a10487136cac9ff9661ead
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-18 16:30:52 -0500

Update openldap scripts to run behind reverse proxy


---
diff --git a/ansible-playbook/group_vars/docker-hosts/networking.yml b/ansible-playbook/group_vars/docker-hosts/networking.yml
index 38e7fdc6b..11afcddab 100644
--- a/ansible-playbook/group_vars/docker-hosts/networking.yml
+++ b/ansible-playbook/group_vars/docker-hosts/networking.yml
@@ -51,3 +51,5 @@ haproxy_port_args:
 - "{{ buildbot_ip_addr }}:443:443"
 - "{{ nexus_ip_addr }}:80:80"
 - "{{ nexus_ip_addr }}:443:443"
+- "{{ phpldapadmin_ip_addr }}:80:80"
+- "{{ phpldapadmin_ip_addr }}:443:443"
